composables/useIdentity.ts

```diff
@@ -27,7 +27,7 @@ export default function useIdentity({
   )
 
   const { data, refetch, loading } = useGraphql({
-    clientName: computed(() => (isDotAddress.value ? 'pid' : 'kid')),
+    clientName: computed(() => (isDotAddress.value ? 'pid' : 'kid')).value,
```

Comment:
I would revert this line and keep it as original.
```
clientName: computed(() => (isDotAddress.value ? 'pid' : 'kid'))
```

Reply:
alright will do

Reply:
OK. We could fix the type warning later because this line would cause a bug.